s3/replication/build-runalltest.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
+
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+#
+# build-runalltest.sh allows to build all multisite moudles,
+# create virtual environment, populating service files with
+# correct paths, copying to default systemd location and
+# starting daemon processes and running load transfer tests for
+# s3replicator and s3replicationmanager.
+#
+# Usage : sh ./build-runalltest.sh
+# 
+
+
+# Project root directory
+MULTISITE_ROOT_DIR="$(cd ../../; pwd)"
+echo "Project directory : "$MULTISITE_ROOT_DIR
+
+# Replication directory
+REPLICATION_DIR="$(pwd)"
+echo "Replication directory :" $REPLICATION_DIR
+
+# Module directory
+S3_REPLICATION_COMMON_SRC_DIR=$REPLICATION_DIR/common/src
+S3_REPLICATION_MANAGER_SRC_DIR=$REPLICATION_DIR/manager/src
+S3_REPLICATOR_SRC_DIR=$REPLICATION_DIR/replicator/src
+
+# Test directory
+S3_REPLICATION_COMMON_TEST_DIR=$REPLICATION_DIR/common/tests/system
+S3_REPLICATION_MANAGER_TEST_DIR=$REPLICATION_DIR/manager/tests/system
+S3_REPLICATOR_TEST_DIR=$REPLICATION_DIR/replicator/tests/system
+
+echo "Module directories -"
+echo "Common module directory :" $S3_REPLICATION_COMMON_SRC_DIR
+echo "Manager module directory :" $S3_REPLICATION_MANAGER_SRC_DIR
+echo "Replicator module directory :" $S3_REPLICATOR_SRC_DIR
+
+# Active an environment
+python3 -m venv $REPLICATION_DIR/s3env
+source $REPLICATION_DIR/s3env/bin/activate
+
+# Check and download dependencies
+pip3 install --upgrade setuptools
+
+pip3 install -r $REPLICATION_DIR/common/requirements.txt
```

Comment:
this is already done in make install for each module

Reply:
done